.changeset/two-rules-melt.md

```diff
@@ -0,0 +1,8 @@
+---
+"@mds-core/mds-schema-validators": minor
+"@mds-core/mds-attachment-service": patch
+"@mds-core/mds-geography-service": patch
+"@mds-core/mds-transaction-service": patch
+---
+
+refactored validators to use ajv instead of joi
```

Comment:
Are these .changeset files the same kind of thing as https://github.com/atlassian/changesets/blob/main/docs/detailed-explanation.md ?

Reply:
> Are these .changeset files the same kind of thing as https://github.com/atlassian/changesets/blob/main/docs/detailed-explanation.md ?

Yup!